src/cryptonote_core/service_node_voting.cpp

```diff
@@ -319,15 +319,22 @@ namespace service_nodes
     // NOTE: Validate vote age
     //
     {
-      uint64_t delta_height = latest_height - vote.block_height;
-      if (vote.block_height < latest_height && delta_height >= VOTE_LIFETIME)
+      // If we get an incoming vote that is outside the acceptable range by this many votes then
+      // just ignore it instead of dropping the connection; the sending side could be a couple
+      // blocks out of sync and sending something that it thinks is legit.
+      constexpr uint64_t VOTE_BUFFER = 5;
+
+      bool height_in_buffer = false;
+      if (latest_height > vote.block_height + VOTE_LIFETIME)
```

Comment:
Can you also add this to the verify_tx_state_change path? You will need to update at the caller's site in blockchain.cpp to not set m_verification_failed and put it in the verify function instead setting it to failed, IFF the vote was not within the buffer or some other part further down failed validation.